src/features/favorites/components/FavoritesResults.tsx

```diff
@@ -20,32 +19,71 @@ import { ColorsEnum, getSpacing, Spacer, TAB_BAR_COMP_HEIGHT } from 'ui/theme'
 
 const keyExtractor = (item: FavoriteResponse) => item.id.toString()
 
-export const FavoritesResults: React.FC = React.memo(() => {
+const useFetchFavorites = (data: PaginatedFavoritesResponse | undefined) => {
```

Comment:
Ce n'est pas un hook, et il n'y pas de fetch : 
```suggestion
function paginateFavorites(data: PaginatedFavoritesResponse | undefined) {
```

Reply:
OK